packages/plugin-ext/src/main/browser/terminal-main.ts

```diff
@@ -179,6 +179,13 @@ export class TerminalServiceMainImpl implements TerminalServiceMain, Disposable
         }
     }
 
+    $setName(id: string, name: string): void {
+        const terminal = this.terminals.getById(id);
+        if (terminal) {
+            terminal.setTitle(name);
+        }
```

Comment:
You've used a different writing for `$setNameByTerminalId` which I think is good:

```suggestion
        this.terminals.getById(id)?.setTitle(name);
```